Cargo.toml

```diff
@@ -64,7 +64,9 @@ version = "=22.0.4"
 # Dependencies from the rs-stellar-rpc-client repo:
 [workspace.dependencies.soroban-rpc]
 package = "stellar-rpc-client"
-version = "=22.0.0"
+#version = "=22.0.0"
+git = "https://github.com/stellar/rs-stellar-rpc-client"
+branch = "get-health"
```

Comment:
I've approved the PR that this branch references:
https://github.com/stellar/rs-stellar-rpc-client/pull/24

If that library has no breaking changes I think it can be released so you can reference a new version here.